File: flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/VLCMediaPlayer.java
package software.solid.fluttervlcplayer;

import android.os.Handler;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import org.videolan.libvlc.MediaPlayer;
import org.videolan.libvlc.interfaces.AbstractVLCEvent;
import org.videolan.libvlc.interfaces.ILibVLC;

class VLCMediaPlayer extends MediaPlayer {

    VLCMediaPlayer(@NonNull ILibVLC libVLC) {
        super(libVLC);
    }

    @Override
    public synchronized void setEventListener(
            @Nullable AbstractVLCEvent.Listener<Event> listener,
            @Nullable Handler handler
    ) {
        super.setEventListener(listener, handler);
    }
}

File: flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/VLCMediaPlayer.kt
@file:JvmName("VLCMediaPlayer")

package software.solid.fluttervlcplayer

import android.os.Handler
import org.videolan.libvlc.MediaPlayer
import org.videolan.libvlc.interfaces.AbstractVLCEvent
import org.videolan.libvlc.interfaces.ILibVLC

class VLCMediaPlayer(libVLC: ILibVLC) : MediaPlayer(libVLC) {
    public override fun setEventListener(
        listener: AbstractVLCEvent.Listener<Event>?, handler: Handler?
    ) = super.setEventListener(listener, handler)
}